fs/layer/node.go

```diff
@@ -221,6 +268,10 @@ func (n *node) readdir() ([]fuse.DirEntry, syscall.Errno) {
 	start := time.Now() // set start time
 	defer commonmetrics.MeasureLatencyInMicroseconds(commonmetrics.NodeReaddir, n.fs.layerDigest, start)
 
+	if n.fs.operationCounter != nil {
+		n.fs.operationCounter.Inc(fuseOpReaddir)
+	}
+
```

Comment:
`fuseOpReaddir` is Inc'ed both in `Readdir` and `readdir`. This will double the count right?

Reply:
Yes, will remove it .